CONTRIBUTING.md

```diff
@@ -46,13 +46,36 @@ See the [contributor's testing guide](https://github.com/kudobuilder/kudo/blob/m
 
 ## Community, Discussion, and Support
 
-Learn how to engage with the Kubernetes community on the [community page](https://kudo.dev/community/).
+Learn how to engage with the KUDO community on the [community page](https://kudo.dev/community/).
 
 You can reach the maintainers of this project at:
 
 - [Slack](https://kubernetes.slack.com/messages/kudo/)
 - [Mailing List](https://groups.google.com/d/forum/kudobuilder)
 
+## Code culture
+
+This is a set of practices we try to live by when developing KUDO. These are just defaults (soft rules). Deviations from them are possible, only has to be justified.
+
+### General guidelines
+- Master is always releasable (green CI)
+- All feature/bug-fixing work should have an open issue with a description
+- Every user-facing feature that is NOT behind a feature gate should have an integration test
+
+### Pull requests
+- One core-team member has to approve the PR to be able to merge (defined in code owners file)
```

Comment:
```suggestion
- One code owner has to approve the PR to be able to merge.
```

Reply:
What is "an owner"? The `OWNERS` file only has `approvers` and `reviewers`.

Reply:
Let's delete OWNERS until when/if we go back to Prow. CODEOWNERS only has the concept of an owner.

Reply:
I think they are still valuable because github assigns PRs based on them. I still feel like they are a meaningful definition of what we consider a "core-team member"

Reply:
I've rephrased it a bit @porridge let me know if that works for you